### DISCUSSION TITLE
How to correlate starting point with another variable like 'coh'?

### DISCUSSION THREAD

**Discussion by DianaY-J** [mwshinn/PyDDM#106] [Ask for help] [answered]
Hi Github folks,

I want to correlate starting point with another variable, just like how you can make drift rate dependent on 'coh', but it gives attribution error like the picture attached below shows. How can I implement this? 

Thank you so much beforehand!

<img width="1358" alt="Screenshot 2024-07-21 at 17 30 56" src="https://github.com/user-attachments/assets/b73e8ced-b2f8-4db1-8161-0ccaae5f70dd">

**Comment by mwshinn** [comment, marked as answer]
You need to specify "a" as a condition for each trial in the sample "mysample", otherwise there is no value to use when doing the fitting.

**Reply by DianaY-J** [reply]
Ohhh you are right! So smart, thank you so much for your efficient respond!!!
